### DISCUSSION TITLE
Make fastapi_users.db.sqlalchemy.GUID JSON-serializable

### DISCUSSION THREAD

**Discussion by slado122** [fastapi-users/fastapi-users#473] [Ideas]
It's an enhancement. You could make GUID type JSON-serializable so it will work out of the box with libraries like Celery and Dramatiq.

**Comment by frankie567** [comment]
Hi @slado122!

Sorry for the response delay. I think I see you want to do here, however, in my opinion, it's better if you work with Pydantic models instead of `sqlalchemy` models. Those DB models are only there to work with the database.

Sending serialized Pydantic models (using `.dict()` or `.json()`) to Celery and deserialize it back into a Pydantic model in the task process sounds more "FastAPI way".

PS : For this kind of suggestion, please favor [Discussions](https://github.com/frankie567/fastapi-users/discussions)
